test/rails/controller_test.rb

```diff
@@ -33,7 +33,7 @@ class ProcessParamsTest < ActionController::TestCase
   end
 
   test "Create with overridden #process_params" do
-    post :create, band: {name: "Kreator"}
+    post :create, band: {name: "Kreator", locality: "Essen"}
```

Comment:
Please revert, this is the whole point of the test to **not** provide `locality`.